src/main/java/com/gmail/goosius/siegewar/tasks/SiegeWarTimerTaskController.java

```diff
@@ -76,6 +74,7 @@ private static void evaluateTimedSiegeOutcome(Siege siege) {
 				//Wait for siege immunity timer to end then delete siege
 				if (System.currentTimeMillis() > TownMetaDataController.getSiegeImmunityEndTime(siege.getDefendingTown())) {
 					SiegeController.removeSiege(siege, SiegeSide.NOBODY);
+					SiegeController.removeSiegedTown(siege);
```

Comment:
This is already being removed from the map in the removeSiege, at lines 194, 195.